rssconfig.py

```diff
@@ -16,7 +16,7 @@ class RssConfig(object):
             ("quality", """480p;720p;1080p""", "480p, 720p or 1080p", "720p"),
             ("pushbulletapi","str","Your Pushbullet-API key",""),
             ("hoster", """OBOOM;Uploaded;Share-Online;Zippyshare""", "Hoster to load from", "Uploaded"),
-            ("historical","bool","Use the search function in order to match older entries","False"),
+            ("historical","bool","Use the search function in order to match older entries","True"),
```

Comment:
Wow, I would've not been able to spot this bug.

Was the True/False part not properly recognized as bool?

I would like to keep the historical setting off by default, but for checking purposes I will merge this PR as requested!